File: TiaCodegen/Commands/Comparisons/Eq.cs
using TiaCodegen.Interfaces;

namespace TiaCodegen.Commands.Comparisons
{
    public class Eq : CompareOperator // ==
    {
        public Eq(params IOperationOrSignal[] operationOrSignals)
            : base(operationOrSignals)
        { }
    }
}

File: TiaCodegen-ts/src/Commands/Comparisons/Eq.ts
import { IOperationOrSignal } from '../../Interfaces/IOperationOrSignal';
import { CompareOperator } from './CompareOperator';

export class Eq extends CompareOperator {
    constructor(...operationOrSignals: IOperationOrSignal[]) {
        super(...operationOrSignals);
    }
}
